konlpy/tag/_twitter.py

```diff
@@ -66,11 +73,30 @@ def phrases(self, phrase):
         """Phrase extractor."""
 
         return [p for p in self.jki.phrases(phrase).toArray()]
-    
+
     def normalize(self, phrase):
         text = self.jki.normalize(phrase)
         return text
 
+    def __spacing(self, sentence: str, pos_result: list):
```

Comment:
https://www.python.org/dev/peps/pep-0484/
It seems also supports version 3.5. sorry